### ISSUE TITLE
(Firefox) Privacy Badger doesn't block trackers on pinned tabs at browser startup

### ISSUE THREAD

**Issue by tkhv** [EFForg/privacybadger#2720]
**What is your browser and browser version?**

Firefox 83.0 (Windows)

**What is broken and where?**

When I launch Firefox and have a few pinned tabs, Privacy badger doesn't block trackers on those tabs at startup, unless I refresh that tab. Other extensions, such as uBlock Origin for example, block ads on pinned tabs just fine at startup. 

I've attempted launching Edge with pinned tabs, and trackers are being blocked by Privacy Badger there, this is an issue only with Firefox AFAIK.

**Comment by ghostwords** [comment]
Thanks for getting in touch! I'm going to mark this as a duplicate of #1845.
